app/helpers/foreman_leapp/template_helper.rb

```diff
@@ -11,5 +18,12 @@ def build_remediation_plan(remediation_ids, host)
                            .map { |r| "#{r['context'].join(' ')}\n" }
                            .join
     end
+
+    def leapp_preupgrade
```

Comment:
instead of extracting this to a macro, can we extract it to separate template? Note that we can easily render it with `<%= render_template 'leapp_preupgrade' %>` and users can still easily modify its content. The `remediation.erb` can then do
```
<%  if input('run_preupgrade') == 'true' -%>
<%=   render_template 'leapp_preupgrade -%>
<%  end -%>
```

also note the == 'true'. I think the input is always string